about.html

```diff
@@ -0,0 +1,104 @@
+              training, prototyping techniques, social events, and networking sessions. 
+              The Hub also has a fully equipped digital library, rapid prototyping services (e.g 3D printing), 
+              mock-up labs, meeting rooms, space for exhibition and events, co-working space, and cafeteria.
+            </p>
+          </div>
+          <div class="event-date col-12 pb-3">
+            <p class="text-center w-50 m-auto pb-3">
+              Please contact us per Email for any further questions
+              about Software Developer's Conference Summit
+              <span class="font-weight-bold pb-4">
+                <u>Abenezermesafint2128@gmail.com</u>
+              </span>
+            </p>
+          </div>
+      </section>
+
+      <!-- logo section-->
+      <section class="container-fluid featured-speakers text-dark bg-white border-bottom">
+        <div class="row pt-1 mb-3">
+          <div class="col-12 pt-3">
+            <h3 class="text-center section-title">Creative-Hub Summit Logo</h3>
+            <hr class="section_title_hr">
+          </div>
+        </div>
+        <div class="row logo-description p-4">
+          <div class="col-12">
+            <p class="text-center">
+            The logo of this summit was decided through the logo competition from 8. June to 7. July. and had a reliable voting system
+            which everyone contributed to.
+            </p>
+          </div>
+          <div class="col-12 mt-4 text-center">
+            <figure class="border p-3">
+              <img class="img-fluid" width="200" src="Design/images/Creative-Hub.svg" alt="Logo">
+            </figure>
+          </div>
+        </div>
+      </section>
+
+      <section class="container-fluid featured-speakers text-dark bg-white border-bottom">
+        <div class="row pt-3 mb-5">
+          <div class="col-12">
+            <h3 class="text-center section-title">Our previous conference summits</h3>
+            <hr class="hr-title">
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-12">
+            <p class="text-center">Take a look at our previous summit</p>
+          </div>
+        </div>
+        <div class="row previous p-4 justify-content-center">
+          <div class="col-12 col-md-5 mt-4 text-center text-white p-5 prev_1">
+            <h3 class="font-weight-bold mb-3">12.July.2020</h3>
+            <p class="font-weight-bold text-white">Software Developers Conference Summit 2020 in Ethiopia</p>
+          </div>
+          <div class="col-12 col-md-5 ml-md-2 mt-4 text-center text-white p-5 prev_2">
+            <h3 class="font-weight-bold mb-3">18.September.2021</h3>
+            <p class="font-weight-bold text-white">Software Developers Conference Summit 2021 in Ethiopia</p>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="shared_footer container-fluid p-4 bg-dark-custom">
```

Comment:
- [ ] Please make sure to align your content in the footer as follows:
- Logo aligned to the left.
- Text aligned to the right.

This is not only for a requirement, but for users, it's better to keep your page looking as organized as possible, and if you keep your footer as it is right now, a user may think something is wrong and think that there is some content missing on the right side of the footer.

Reply:
Ok I will aligned the logo and text to left. Thanks for suggestion.